src/appsettings/settings.py

```diff
@@ -986,6 +1110,8 @@ def __init__(
         validators=(),
         key_type=None,
         value_type=None,
+        outer_delimiter=" ",
```

Comment:
Having the values separated by a single space, but not multiple or other blank values seems rather unexpected.

Reply:
Default value changed to `None`, which is (according to python docs) any whitespace sequence.